src/rma/destruction/templates/destruction/recordmanager_list.html

```diff
@@ -7,24 +8,51 @@ <h1 class="title">Mijn vernietigingslijsten</h1>
 </header>
 
 <article class="destruction-list">
-    <section class="destruction-list__content content-panel">
+    <section class="destruction-list__content">
         <header class="destruction-list__header">
-            <h2 class="destruction-list__title section-title section-title--highlight">Vernietigingslijsten</h2>
             <nav class="destruction-list__nav">
                 <a class="btn" href="{% url 'destruction:record-manager-create' %}">Opstellen</a>
             </nav>
         </header>
 
-
         {% for object in object_list %}
-            <div class="list-item {% if object.assignee == request.user %}list-item--important {% endif %}">
-            <a class="list-item__title" href="#">{{ object.name }}</a>
```

Comment:
`.list-item` is also used in notification list. I think it should also be removed there

Reply:
Replaced in 09cb5c8ba47b34b4dd7c5493799b27f4db0725bb